src/ripple/ledger/impl/View.cpp

```diff
@@ -387,6 +388,37 @@ cdirNext (BasicView const& view,
     return true;
 }
 
+enabledAmendments_t
+getEnabledAmendments (BasicView const& view)
+{
+    enabledAmendments_t amendments;
+    auto const sleAmendments = view.read(keylet::amendments());
+
+    if (sleAmendments)
+    {
+        for (auto const &a : sleAmendments->getFieldV256 (sfAmendments))
```

Comment:
The word "amendment" appears NINE times in this diff! Suggested:
`auto const sle = view.read(keylet::amendments());`